xml/obs_source_service.xml

```diff
@@ -7,154 +7,299 @@
-    already contains the same file, the newly generated file must be dropped.
-   </para>
-  </listitem>
-  <listitem>
-   <para> local services can be added and used by everybody. </para>
-  </listitem>
-  <listitem>
-   <para> server side services must be installed by the admin of the OBS
-    server. </para>
-  </listitem>
-  <listitem>
-   <para> service can be defined per package or project wide. </para>
-  </listitem>
- </itemizedlist>
- <sect1>
-  <title>Using Services for Validation</title>
-  <para> Source Services may be used to validate sources. This can happen per
-   package, which is useful when the packager wants to validate that downloaded
-   sources are really from the original maintainer. Or validation can happen
-   for an entire project to apply general policies. These services cannot get
-   skipped in any package </para>
-  <para> Validation can happen by validating files (for example using the
-    <command>verify_file</command> or <command>source_validator</command>
-   service. These services just fail in the error case which leads to the build
-   state "broken". Or validation can happen by redoing a certain action and
-   store the result as new file as <command>download_files</command> is doing.
-   In this case the newly generated file will be used instead of the committed
-   one during build. </para>
+
+ <sect1 xml:id="sec.obs.sserv.about">
+  <title>About Source Service</title>
+  <para>
+   Source Services are tools to validate, generate or modify sources in a
+   trustable way. They are designed as smallest possible tools and can be
+   combined following the powerful idea of the classic UNIX design.
+  </para>
+  <para>
+   Design goals of source services were:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para> server side generated files must be easy to identify and must not be
+     modifiable by the user. This way others user can trust them to be
+     generated in the documented way without modifications. </para>
+   </listitem>
+   <listitem>
+    <para> generated files must never create merge conflicts </para>
+   </listitem>
+   <listitem>
+    <para> generated files must be a separate commit to the user change </para>
+   </listitem>
+   <listitem>
+    <para> services must be runnable at any time without user commit </para>
+   </listitem>
+   <listitem>
+    <para> services must be runnable on server and client side in the same way
+    </para>
+   </listitem>
+   <listitem>
+    <para> services must be designed in a safe way. A source checkout and
+     service run must never harm the system of a user. </para>
+   </listitem>
+   <listitem>
+    <para> services shall be designed in a way to avoid unnecessary commits.
+     This means there shall be no time-dependent changes. In case the package
+     already contains the same file, the newly generated file must be dropped.
+    </para>
+   </listitem>
+   <listitem>
+    <para> local services can be added and used by everybody. </para>
```

Comment:
add also the "buildtime" services here: Services running local or inside the build environment can get created, added and used by everbody.

Reply:
Ok, used your 2nd sentence.